src/app/zap-templates/zcl/data-model/chip/global-enums.xml

```diff
@@ -60,6 +60,13 @@ TODO: Make these structures global rather than defining them for each cluster.
     <item name="Equipment" value="0x02"/>
   </enum>
 
+  <enum name="StreamUsageEnum" type="enum8">
```

Comment:
Why this random move inside the file?  Is Alchemy output unstable, or is something else going on here?

Reply:
Please check with @hasty to figure out what's going on here,